ibmdbpy/frame.py

```diff
@@ -2316,3 +2323,106 @@ def check_numeric_columns(idaobject):
         check_numeric_columns(self)
         if isinstance(other, ibmdbpy.IdaSeries)|isinstance(other, IdaDataFrame):
             check_numeric_columns(other)
+
+###############################################################################
+### New functionalities for IdaDataFrames -- Experimental
+###############################################################################
+    
+    def delete_na(self, columns, logic = "any", inplace = False):
+        """
+        
+        Parameters
+        -------
+        columns: list of eligible column names
+        logic: str, optional, "any" by default. If logic is set to "any" then all rows which contain a NaN value in any 
+        (id est at least one) of the cited columns is deleted, this is a union condition; 
+        if logic is set to "all", then only rows containing null values in all the fields are deleted, this is an intersection condition.         
+        inplace: bool, optional, False by default. If True, then the underlying table is physically modified,
+        if False, then the original objects remain unaffected, a copy of the IdaDataFrame is made and modified. 
+        
+        Returns
+        -------
+        If inplace is True, the original table is modified and the IdaDataFrame will be modified accordingly. No return.
+        If inplace is False, a new IdaDataFrame is returned, the original table is not affected. This new IdaDataFrame points to a 
```

Comment:
This should mention what the name of the new table is - imagine a user is running this code on a system with hundreds or thousands of tables. You don't want them to manually figure out the new table name.

Reply:
Yes sure. I added the following to be explicit:
`print('A new table with filtered rows is available under the name %s.' %tablename)
print('The newly created IdaDataFrame refers to this new table.')`